packages/pages/src/common/src/template/types.ts

```diff
@@ -98,6 +98,24 @@ export interface TemplateConfig {
   alternateLanguageFields?: string[];
   /** The name of the onUrlChange function to use. */
   onUrlChange?: string;
+  /** Locales for a static page */
+  locales?: string[];
```

Comment:
I don't think we want this. A templateConfig (which now represents and entity template) doesn't have this option. Locales exist in the Stream definition.

Reply:
I see yes, that makes sense.  Both `TemplateConfig` and `StaticTemplateConfig` both eventually get turned into `TemeplateConfigInternal` which does have `locale`, and that is passed through and used thought Pages.  `TemplateConfig` is just a user facing interface. Removed `locale` from `TemplateConfig`!

Reply:
Precisely.